app/src/main/java/nu/yona/app/ui/BaseActivity.java

```diff
@@ -34,35 +32,32 @@
 	private InputMethodManager inputMethodManager;
 	private Runnable backPressListener = null;
 	private PauseResumeHook hook;
+	private static int progressDialogCount = 0;
+
+
+	public void toggleLoadingView(boolean loading)
+	{
+		toggleLoadingView(loading, null);
+	}
 
 	/**
 	 * Show loading view.
 	 *
 	 * @param loading the loading
 	 * @param message the message
 	 */
-	public void showLoadingView(boolean loading, String message)
+
+	public void toggleLoadingView(boolean loading, String message)
```

Comment:
The method name is better, but it's still wrong. Toggle suggests you're hiding if it it's visible and showing it when it's not visible.

Given that you've removed all logic (which was a good action!), just change the callers: make then call ``displayLoadingView`` or ``dismissLoadingView`` depending on their needs.